src/index.js

```diff
@@ -28,10 +31,15 @@ const client = new ApolloClient({
 });
 
 ReactDOM.render(
-    <ApolloProvider client={client}>
-        <App />
-    </ApolloProvider>
+    <Provider store={store}>
+        {/* remove appollo provider? */}
```

Comment:
For the sake of simplicity, it's probably best to forego Apollo and GraphQL for this project, and stick to basic Redux and REST, instead.